transitionable.gemspec

```diff
@@ -36,5 +36,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_development_dependency "rspec", "~> 3.7"
   spec.add_development_dependency 'pry'
-  spec.add_dependency "activesupport", "~>5.2"
```

Comment:
how about `"> 5.2"`? otherwise monolith can use any future release of this library until it upgrades to rails 6